topics/ecology/tutorials/openrefine_gbif/tutorial.md

```diff
@@ -0,0 +1,327 @@
+---
+layout: tutorial_hands_on
+
+title: Cleaning GBIF data using OpenRefine
+zenodo_link: ''
+questions:
+- How can I use OpenRefine to clean data?
+- How do I check and clean biodiversity data using OpenRefine?
+
+objectives:
+- Use OpenRefine faceting functionnalities to apply mass editing and manage duplicates
+- Use OpenRefine clustering and filtering functionnalities to edit, transform data notably using regular expression
+- Use OpenRefine to apply API services on your data
+time_estimation: 0h45m
+key_points:
+- OpenRefine is a powerfull tool, with many functionnalities to check, clean and enrich your data
+contributions:
+    authorship:
+        - yvanlebras
+        - sophiepamerlon
+        - laurarussell
+        - gbiftraining
+    funding:
+        - pndb
+
+
+---
+
+
+In this tutorial we will use OpenRefine tool to clean occurence records retrieved from GBIF.
+
+This tutorial is based on the GBIF {% cite GBIFtraining2021 %} "Biodiversity data mobilization course" accessible on [GBIF website](https://docs.gbif.org/course-data-mobilization/en/) on Use case 1 ["Herbarium Specimens"](https://docs.gbif.org/course-data-mobilization/en/exercise-3a-c.html) using file from Exercice 3C, at the bottom of the page, reachable through this direct [URL](https://docs.gbif.org/course-data-mobilization/exercise-data/UC1-3c-open-refine.csv).
+
+> <agenda-title></agenda-title>
+>
+> In this tutorial, we will cover:
+>
+> 1. Import data to OpenRefine
+> 2. Faceting
+> 3. Filtering
+> 4. Clustering
+> 5. API use
```

Comment:
The agenda will be autogenerated based on your sections, please don't manually edit. If you want the agenda to look a certain way, may your sections reflect this structure (e.g. all 1st and 2nd level, so `#` and `##` will show up in the agenda)

```suggestion

```